.gitignore

```diff
@@ -35,3 +35,6 @@ yarn-error.log*
 
 # vscode
 .vscode
+yarn.lock
```

Comment:
You might've to clear the `INDEX` before pushing the commits once again else you'll keep pushing the `yarn.lock` file regardless of the contents of the `.gitignore` file. You can refer to this article - https://www.git-tower.com/learn/git/faq/ignore-tracked-files-in-git to know more about how its done.